test/test_torch.py

```diff
@@ -85,14 +85,28 @@ def test_horovod_reinit(self):
 
         hvd.init()
         rank, size = hvd.rank(), hvd.size()
-
         hvd.shutdown()
         hvd.init()
         rank2, size2 = hvd.rank(), hvd.size()
 
         assert rank == rank2
         assert size == size2
 
+    def test_horovod_is_initialized(self):
+        """Test that is_initialized returned by hvd.is_initialized() is correct."""
+        hvd.init()
+        assert hvd.is_initialized()
+
+        gloo_rank = int(os.getenv('HOROVOD_RANK', -1))
+        is_mpi = gloo_rank == -1
+        if is_mpi:
+            # Only applies for Gloo
```

Comment:
Can you change this comment to make it more clear:

```
# Horovod cannot be re-initialized after shutdown when using MPI, so this test can only be done using the Gloo controller
```

Reply:
Sure !